CMakeLists.txt

```diff
@@ -567,7 +567,7 @@ if (GLFW_INSTALL)
 
     install(FILES "${GLFW_BINARY_DIR}/src/glfw3Config.cmake"
                   "${GLFW_BINARY_DIR}/src/glfw3ConfigVersion.cmake"
-            DESTINATION lib${LIB_SUFFIX}/cmake/glfw)
+            DESTINATION ${GLFW_CONFIG_PATH})
 
     install(EXPORT glfwTargets DESTINATION lib${LIB_SUFFIX}/cmake/glfw)
```

Comment:
It may be appropriate to change glfwTargets DESTINATION to ${GLFW_CONFIG_PATH} as well.